src/libs/ReportUtils.js

```diff
@@ -4120,6 +4122,29 @@ function getParticipantsIDs(report) {
     return participants;
 }
 
+/**
+ * Returns an array of the visible member accountIDs for a report
+ *
+ * @param {Object} report
+ * @returns {Array}
+ */
+function getVisibleMemberIDs(report) {
+    if (!report) {
+        return [];
+    }
```

Comment:
NAB, but what's the case where this would be called with no report? Can we alternatively just not call it in that situation? This possibly indicates a silent failure somewhere, but maybe it's expected that the report doesn't exist?

Reply:
I can't find a place where this function is called that a report would not exist:
ReportDetailsPage
ReportParticipantsPage
ShareCodePage (for IOU, but here we check that the report exists before calling)

But every other function that gets called before `getVisibleMemberIDs` handles the report as nullable, so I am not confident in removing it, even though I tested without and and it seems to work fine.

Reply:
Maybe when we are done with the TS migration we will be able to easily do this kinds of refactors?